zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSetting.java

```diff
@@ -716,6 +719,11 @@ private String getInterpreterClassFromInterpreterSetting(String replName) {
         return info.getClassName();
       }
     }
+    //TODO(zjffdu) It requires user can not create interpreter with name `conf`,
+    // conf is a reserved word of interpreter name
+    if (replName.equals("conf")) {
```

Comment:
why do we have to hardcode this?

Reply:
I make `conf` as a reserved interpreter name for this generic `ConfInterpreter` so that all the interpreters can use it. e.g. `%md.conf`, `%sh.conf`, `%spark.conf`